src/emojify.ts

```diff
@@ -21,9 +21,9 @@ export const emojify = (
   const fallbackFunction =
     fallback === undefined ? fallback : asFunction(fallback)
 
-  assert.string(input)
-  assert.any([is.undefined, is.function], fallbackFunction)
-  assert.function(format)
+  is.assert.string(input)
+  is.assert.any([is.default.undefined, is.default.function_], fallbackFunction)
```

Comment:
I played around with these a bit but couldn't find anything cleaner. Ran out of time today. But not super please about using a `.default.` 🤔